### DISCUSSION TITLE
ad hoc filter variable configuration: determine which operators are presented

### DISCUSSION THREAD

**Discussion by darrenjaneczek** [grafana/grafana#61591] [Dashboards, Panels & Visualizations]
Problem:
Our datasource is able to handle all ad hoc filter operators with the exception of `<` and `>`.
A user is free to select these, but it results in query errors.

Proposed solution:
Add a configuration section for variables of type "ad hoc filters," which allows you to select operators to omit from the list.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82865. Feel free to continue the discussion around this there. Thank you!
